libraries/AP_HAL_PX4/RCInput.cpp

```diff
@@ -111,6 +119,30 @@ void PX4RCInput::clear_overrides()
     }
 }
 
+const char *PX4RCInput::input_source_name(uint8_t id) const
+{
+    switch(id) {
+    case input_rc_s::RC_INPUT_SOURCE_UNKNOWN:         return "UNKNOWN";
+    case input_rc_s::RC_INPUT_SOURCE_PX4FMU_PPM:      return "PX4FMU_PPM";
```

Comment:
@muramura That would certainly be the case!  However, that was mechanically translated from the place the definitions for the input sources are defined, and that may make for easier debugging and/or replication of the translation.